api/v6/api.go

```diff
@@ -28,6 +28,7 @@ const (
 	ReadOnlySystem   ReadOnlyReason = "System"
 	ReadOnlyNoRepo   ReadOnlyReason = "NoRepo"
 	ReadOnlyNotReady ReadOnlyReason = "NotReady"
+	ReadOnlyROMode   ReadOnlyReason = "ReadonlyMode"
```

Comment:
Nitpick:

```suggestion
	ReadOnlyROMode   ReadOnlyReason = "ReadOnlyMode"
```

Reply:
I went back and forth on this many times.  there are examples of both in the wild.  Both are used in the Go codebase about evenly.  Both variants will feel weird to different people at for different reasons (if not simultaneously).